lib/takeaway.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'messager'
+require_relative 'menu'
+
+class TakeAway
+  
+  attr_reader :read_menu, :menu, :orders, :messager      
+  
+  def initialize(menu = Menu.new, messager = Messager.new)
+    @menu_obj = menu
+    @messager = messager
+    @orders = []
+    @menu =  @menu_obj.menu
+  end
+  
+  def change_menu(file)
+    @menu_obj = Menu.new(file)
+  end
+
+  def read_menu
+    @menu_obj.read_menu
+  end
+  
+  def order(item, qty=1)
+    print :"That item is not on the menu.\n" unless @menu_obj.contains?(item) 
+    (qty.times {add_item(item)}) if @menu_obj.contains?(item)
```

Comment:
Why keep repeating the conditional `if @menu_obj.contains?(item)`.  You've already tested it once in the method above, so just prevent the method executing any further